front-end/yarn.lock

```diff
@@ -14792,19 +14666,6 @@ tapable@^1.0.0, tapable@^1.1.3:
   resolved "https://registry.yarnpkg.com/tapable/-/tapable-1.1.3.tgz#a1fccc06b58db61fd7a45da2da44f5f3a3e67ba2"
   integrity sha512-4WK/bYZmj8xLr+HUCODHGF1ZFzsYffasLUgEiMBY4fgtltdO6B4WJtlSbPaDTLpYTcGVwM2qLnFTICEcNxs3kA==
 
-tar@^4.4.2:
```

Comment:
ah, and why all those yarn.lock changes.. idk, I keep delete the `yarn.lock` and do `yarn` again quite regularly as this is what helped us reduce duplication quite a lot last time..

Reply:
done